pkg/event/processor.go

```diff
@@ -129,6 +127,8 @@ func NewBatchEventProcessor(options ...BPOptionConfig) *BatchEventProcessor {
 		opt(p)
 	}
 
+	p.logger = logging.GetLogger(p.sdkKey, "BatchEventProcessor")
```

Comment:
no, because the sdk key is part of the config options.  So, first I run my config options, they, I grab the sdk key, then I get the logger with the sdk key.